scala/private/phases/phase_compile.bzl

```
@@ -261,6 +261,7 @@ def _build_nosrc_jar(ctx):
     zipper_arg_path = ctx.actions.declare_file("%s_zipper_args" % ctx.label.name)
     ctx.actions.write(zipper_arg_path, resources)
     cmd = """
+set -e
```

```suggestion
set -o errexit
set -o nounset
set -o pipefail
```

Generally I prefer long options for scripting. If we're adding `-e`, might as well add `nounset` and `pipefail` as well? I'd be curious if `touch` always succeeds. It might not be rules_scala, but I sometimes see `touch` getting rejected. (Likely it already exists)